### DISCUSSION TITLE
绘制组件中怎么获取所有实例？

### DISCUSSION THREAD

**Discussion by yangaphero** [zouyaoji/vue-cesium#381] [General]
getDrawingActionInstances方法返回的数据中怎么能够找到实例的postion等信息？
![image](https://user-images.githubusercontent.com/29125379/233894210-9eed552a-5175-4de6-8232-043d3ebb6857.png)

**Comment by zouyaoji** [comment]
https://github.com/zouyaoji/vue-cesium/discussions/206

参考一下 取到`renderDatas`
